src/Lumi/Components2/Slat.purs

```diff
@@ -0,0 +1,84 @@
+module Lumi.Components2.Slat
+  ( SlatProps
+  , SlatInteraction
+  , slat
+  , _interactive
+  , module Styles.Slat
+  ) where
+
+import Prelude
+import Data.Maybe (Maybe(..))
+import Data.Newtype (un)
+import Effect (Effect)
+import Effect.Unsafe (unsafePerformEffect)
+import Lumi.Components (LumiComponent, PropsModifier, lumiComponent, propsModifier)
+import Lumi.Styles (styleModifier_, toCSS)
+import Lumi.Styles.Slat (_interactive, slat) as Styles.Slat.Hidden
+import Lumi.Styles.Slat hiding (_interactive, slat) as Styles.Slat
+import Lumi.Styles.Theme (useTheme)
+import React.Basic.DOM as R
+import React.Basic.DOM.Events (capture_)
+import React.Basic.Emotion as E
+import React.Basic.Hooks (JSX)
+import React.Basic.Hooks as React
+import Web.HTML.History (URL(..))
+
+type SlatProps
+  = ( content :: Array JSX
+    , interaction :: Maybe SlatInteraction
+    , isList :: Boolean
```

Comment:
This prop doesn't seem to be used anywhere?

Reply:
ah, not anymore, thanks